release.sh

```diff
@@ -107,6 +80,22 @@ sed -e "s/@RELEASE@/${RELEASE}/g" "${ROOTDIR}/nexus-repositories.yaml" | generat
 # sync container images
 skopeo-sync "${ROOTDIR}/docker/index.yaml" "${BUILDDIR}/docker"
 
+# scan container images
+if [[ -z "$NO_SKYK_SCAN" ]]; then
+    mkdir -p "$SNYK_SCAN_DIR"
+    all_images=$(list-images "${ROOTDIR}/docker/index.yaml")
+    cd "$SNYK_SCAN_DIR"
+    for image in $all_images; do
+        snyk-scan "$image"
+    done
+    snyk_results=$(find "$SNYK_SCAN_DIR" -name "snyk.json")
+    snyk-aggregate-results "$snyk_results"
+    for result in $snyk_results; do
+        snyk-to-html "$result"
+    done
+    cd -
```

Comment:
This is pretty neat, I did not know that `cd -` has this behavior!

Reply:
I tend to see `pushd` and `popd` used more frequently in scripts. Seems like one advantage would be that if there were any intermediate `cd`s this way would break, but the `pushd` and `popd` approach is more explicit about which directories to record on the stack.